lib/twitter_cldr/js/mustache/implementation/numbers/numbers.coffee

```diff
@@ -106,7 +123,7 @@ class TwitterCldr.DecimalFormatter extends TwitterCldr.NumberFormatter
           result = transliterator.transliterate(result)
       result
     catch error
-      number
+      number.toString()
```

Comment:
Previously was just returning the number itself, making the return type `string | number` instead of just `string`